src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -59,7 +59,8 @@ private static ArgumentParser getArgumentParser() {
                 .metavar("LOCATION")
                 .help("The GitHub URL or disk locations to clone repository.");
 
-        mutexParser.addArgument("-view")
+        parser.addArgument("-view")
+                .nargs("*")
                 .metavar("PATH")
                 .type(new ReportFolderArgumentType())
                 .help("Starts a server to display the dashboard in the provided directory.");
```

Comment:
Should update this as well (same as what you wrote in the DeveloperGuide)

Reply:
Do you mean update the help text?

Reply:
>Do you mean update the help text?

Yep